x/distribution/types/params.go

```diff
@@ -26,8 +26,8 @@ func ParamKeyTable() paramtypes.KeyTable {
 func DefaultParams() Params {
 	return Params{
 		CommunityTax:        sdk.NewDecWithPrec(2, 2), // 2%
-		BaseProposerReward:  sdk.NewDecWithPrec(1, 2), // 1%
-		BonusProposerReward: sdk.NewDecWithPrec(4, 2), // 4%
+		BaseProposerReward:  sdk.ZeroDec(),            // deprecated
+		BonusProposerReward: sdk.ZeroDec(),            // deprecated
```

Comment:
I would have expected that these two can simply be deleted as we do not need to be backwards compatible at all. But I guess for the sake of minimal changes to the upstream SDK, this is the more reasonable approach still.

Q: These would still be changeable via governance though?

Reply:
> These would still be changeable via governance though?

yes, this is only changing the defaults

Reply:
We can delete the params if that's preferred. I set the params to zero b/c the PR in the description above and the issue @liamsi created:

> In case we launch with 0.46, we should set the param to 0